package.json

```diff
@@ -38,13 +42,11 @@
     "got": "^11.6.2",
     "prettier": "2.1.1",
     "prom-client": "^12.0.0",
+    "router": "^1.3.5",
```

Comment:
This is not a dev dependency - it is used for the actual library code, not just tests

Reply:
fixed this. added back to dependency list.